docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='environment-authoring-ha-proc']
+= Deploying a high-availability authoring environment
```

Comment:
I see this topic is reused elswhere. I assume that it's just included multiple times, but if the content itself within this file is reused, please note my corrections here for anywhere else they might apply.